packages/ses/src/make-function-constructor.js

```diff
@@ -54,7 +54,7 @@ export function makeFunctionConstructor(realmRec, globaObject, options = {}) {
       value: () => 'function Function() { [native code] }',
       writable: false,
       enumerable: false,
-      configurable: true,
+      configurable: true, // TODO why?
```

Comment:
This is a rider in this pork diff. I intend to remove it, but I don’t want to forget to ask: why do we make function constructors with a configurable `toString`?

Reply:
I don't remember discussing this one specifically, but in general: So that post-repair pre-freeze vetted shims see the normal degree of mutability of JS primordials. It is only the hardening of the primordials that removes all this mutability.

In fact, this should also be `writable: true`, as that is the normal initial JS state. The fact that this one was `writable: false` has me suspicious there's something else going on besides this general rule.